src/constants/Posts.ts

```diff
@@ -0,0 +1,4 @@
+export enum Posts {
+  new = 4,
+  initial = 5
```

Comment:
```suggestion
  initial _amount
```

it;'s not descriptive in the file when it's used